logfire/integrations/pydantic.py

```diff
@@ -292,61 +291,68 @@ def get_schema_name(schema: CoreSchema) -> str:
 class LogfirePydanticPlugin:
     """Implements a new API for pydantic plugins.
 
-    Patches pydantic to accept this new API shape.
+    Patches Pydantic to accept this new API shape.
 
     Set the `LOGFIRE_DISABLE_PYDANTIC_PLUGIN` environment variable to `true` to disable the plugin.
```

Comment:
```suggestion
    Set the `LOGFIRE_DISABLE_PYDANTIC_PLUGIN` or `PYDANTIC_DISABLE_PLUGINS` environment variable to `true` to disable the plugin.
```

Reply:
The reason I didn't do this, is because I'd have to explain that `PYDANTIC_DISABLE_PLUGINS` disables all the plugins. I don't think it's necessary to do it here.